chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/RoomService.kt

```diff
@@ -320,28 +316,28 @@ internal class RoomService(
         }
     }
 
-    fun distributeGlobalEvent(event: ChatEvent) {
-        // This function must map events which we wish to report at room scope that
-        // are not received at room scope from the backend.
-        // Be careful, if you map an event which originated here, you will create
-        // an infinite loop consuming that event.
-        when (event) {
-            is ChatEvent.RoomUpdated ->
-                roomConsumers[event.room.id]?.invoke(RoomEvent.RoomUpdated(event.room))
-            is ChatEvent.RoomDeleted ->
-                roomConsumers[event.roomId]?.invoke(RoomEvent.RoomDeleted(event.roomId))
-            is ChatEvent.UserJoinedRoom ->
-                roomConsumers[event.room.id]?.invoke(RoomEvent.UserJoined(event.user))
-            is ChatEvent.UserLeftRoom ->
-                roomConsumers[event.room.id]?.invoke(RoomEvent.UserLeft(event.user))
-            is ChatEvent.PresenceChange ->
-                roomConsumers.keys.forEach { roomId ->
-                    if (roomStore[roomId]?.memberUserIds?.contains(event.user.id) == true) {
-                        roomConsumers[roomId]?.invoke(
-                                RoomEvent.PresenceChange(event.user, event.currentState, event.prevState)
-                        )
-                    }
-                }
-        }
-    }
+//    fun distributeGlobalEvent(event: ChatEvent) {
```

Comment:
distributeGlobalEvent can be definitely removed

Reply:
RoomService has now been deleted, so this change is no longer necessary.